source/voice_control/assist_create_open_ai_personality.markdown

```diff
@@ -16,63 +16,63 @@ related:
        - Assist can only control {% term entities %} that are [exposed](/voice_control/voice_remote_expose_devices/) to it.
-   - Define the **Model**:
-     - If you have a paid OpenAI subscription that supports GPT4.0, you can select **Submit**.
-     - If you use the free trial version, deselect the **Recommended model settings** checkbox and select **Submit**.
-     - Then, under **Model**, enter `gpt-3.5-turbo` and select **Submit**.
-  
-      ![Add prompt for Mario personality](/images/assist/assistant-openai-mario-09.png)
-  
-3. Give your personality a name:
-   - Select **Rename** and change the name to `OpenAI Mario`.
-  
-      ![Give your Mario personality a name](/images/assist/mario_rename.png)
-4. Create a Mario assistant:
-   - Under {% my voice_assistants title="**Settings** > **Voice assistants**" %}, select **Add assistant**.
-   - Give it a name, select a language, and under **Conversation agent**, select the Mario OpenAI Conversation integration.
-   ![Add a new assistant](/images/assist/assistant-openai-mario-04.png)
-   - Leave the other settings unchanged and select **Create**.
-5. You can repeat this with other OpenAI personalities. You can add as many OpenAI Conversation integrations as you would like.
-   - To add a new personality, you need to create a new API key. Then, add a new OpenAI Conversation integration with that API key.
+             ![Agent with recommended model settings](/images/assist/agent-recommended-model-settings.png)
+
+   - Once your Assist agent has been created, you can go to **Voice assistants** and the three dots menu of your personality, and define if you want Home Assistant's model to be the priority response, and therefore Assist would prefer to handling commands locally .
+             ![Fallback toggle](/images/assist/fallback-assist-toggle.png)
+
+      - If you keep this option selected, if the intent can be answered by Home Assistant it will. It will not have the personality, but the response will be fast and efficient (since it doesn't require to go through the LLM). This is recommended in cases where you can accept not having the IA character reply sometimes and would rather your lights are turned on faster.
+      - If you deselect the option, all the intents will go through the agent. This is recommended when efficiency is not an issue and you need the agent never to break character (for example if your Assist personality is Santa Claus).
+
+
+5. You can uncheck Recommended model settings, hit Submit and it will unblock extra customization. In the specific example of OpenAI, [here](/integrations/openai_conversation/#model) a brief summary of the other settings.
+6. You can test the agent directly from the Voice assistants panel, selecting Start a conversation from the agent's menu. It will control your Home assistant and reply exactly as it will do with any voice hardware.
```

Comment:
```suggestion
6. You can test the agent directly from the Voice assistants panel, selecting Start a conversation from the agent's menu. It will control your Home Assistant and reply exactly as it will do with any voice hardware.
```